hash/megadriv.xml

```diff
@@ -9430,12 +9464,12 @@ Unsupported [Menacer] peripheral
 
 <!-- Only Jpn is confirmed -->
 	<software name="wcsb" cloneof="wcup90">
-		<description>World Championship Soccer (USA) ~ World Cup Soccer (Japan)</description>
+		<description>Soccer (USA) ~ World Cup Soccer (Japan)</description>
 		<year>1989</year>
 		<publisher>Sega</publisher>
 		<info name="serial" value="G-4009 (JPN)"/>
 		<info name="release" value="19890729 (JPN)"/>
-		<info name="alt_title" value="ワールドカップサッカー"/>
+		<info name="alt_title" value="World Championship Soccer / ワールドカップサッカー"/>
```

Comment:
This is legit a mix up, IF `wcsb` shows Soccer as title screen then this is a prototype, cfr. https://segaretro.org/World_Cup_Soccer#Versions

Reply:
It's the Japanese version, before the overseas title was decided. So it just shows "Soccer".

Reply:
Or, a later rev under `Sega Soccer`

Reply:
Actually it's pretty easy to tell: the original version has real players from Mexico '86 lists.

Reply:
I'm going by this: https://tcrf.net/World_Championship_Soccer_(Genesis)

Reply:
By what? Still doesn't resolve what I've said in my original comment. If the title screen is `Sega Soccer` then this is a later rev and needs fixing in description. And while at it, fix it for all.

Reply:
It doesn't say Sega Soccer, it says Soccer
![image](https://github.com/mamedev/mame/assets/47050710/d1953233-2c0e-49e9-b3a9-f2fb7069e5f4)

Reply:
It's REV00 then, and needs to be reflected in the SW list.
Again: I don't see why I'm wasting my time telling you instead of fixing this myself, it doesn't need 8+ comments, it's trivial.

Reply:
Revision 0 means original release, no revisions yet, no need to put that in every version that's not a revision...

Reply:
Oh well, I guess I have to look at this, deal with the new conflict soon.